R/ari_spin.R

```diff
@@ -1,30 +1,23 @@
-#' Create a video from images and text
+#' Generate video from images and text
 #'
 #' Given equal length vectors of paths to images (preferably \code{.jpg}s
 #' or \code{.png}s) and strings which will be
-#' synthesized by
-#' \href{https://aws.amazon.com/polly/}{Amazon Polly} or
-#' any other synthesizer available in
-#' \code{\link[text2speech]{tts}}, this function creates an
+#' synthesized by a text-to-speech engine, this function creates an
 #' \code{.mp4} video file where each image is shown with
 #' its corresponding narration. This function uses \code{\link{ari_stitch}} to
 #' create the video.
 #'
-#' This function needs to connect to
-#' \href{https://aws.amazon.com/}{Amazon Web Services} in order to create the
-#' narration. You can find a guide for accessing AWS from R
-#' \href{http://seankross.com/2017/05/02/Access-Amazon-Web-Services-in-R.html}{here}.
-#' For more information about how R connects
-#' to Amazon Polly see the \code{aws.polly} documentation
-#' \href{https://github.com/cloudyr/aws.polly}{here}.
-#'
 #' @param images A vector of paths to images.
 #' @param paragraphs A vector strings that will be spoken by Amazon Polly.
 #' @param output A path to the video file which will be created.
 #' @param voice The voice you want to use. See
 #' \code{\link[text2speech]{tts_voices}} for more information
 #' about what voices are available.
-#' @param service speech synthesis service to use,
+#' @param model_name (Coqui TTS only) Deep Learning model for Text-to-Speech
+#'   Conversion
+#' @param vocoder_name (Coqui TTS only) Voice coder used for speech coding and
+#'   transmission
```

Comment:
I would remove any args that are engine specific, but the place to document this is in the docs for `coqui_args()`.